src/axom/sidre/core/Group.cpp

```diff
@@ -37,6 +37,48 @@ namespace sidre
 // support path syntax.
 const char Group::s_path_delimiter = '/';
 
+////////////////////////////////////////////////////////////////////////
+//
+// Utility functions to cast ItemCollections to (named) MapCollections
+//
+////////////////////////////////////////////////////////////////////////
+
+MapCollection<View>* Group::getNamedViews()
+{
+  SLIC_ASSERT_MSG(
+    this->isUsingMap(),
+    "Attempting to access named view collection of a (nameless) list Group");
```

Comment:
I find this message and the ones below somewhat unclear. In particular, the "of a (nameless) list Group" part. Thoughts on making it clearer?

Reply:
Reworded